frontend/pages/policies/ManagePoliciesPage/ManagePoliciesPage.tsx

```diff
@@ -215,34 +347,131 @@ const ManagePolicyPage = (): JSX.Element => {
           )}
         </div>
         <div className={`${baseClass}__description`}>
-          <p>Policy queries report which hosts are compliant.</p>
-        </div>
-        {policies && policies.length !== 0 && !isLoadingError && (
-          <InfoBanner className={`${baseClass}__sandbox-info`}>
+          {isPremiumTier && !!selectedTeamId && (
             <p>
-              Your policies are checked every <b>{updateInterval.trim()}</b>.
-              Check out the Fleet documentation on{" "}
-              <a href={DOCS_LINK} target="_blank" rel="noreferrer">
-                <b>how to edit this frequency</b>
-              </a>
+              Add additional policies for <b>all hosts assigned to this team</b>
               .
             </p>
-          </InfoBanner>
-        )}
-        <div>
-          {renderTable(
-            policies,
-            isLoading,
-            isLoadingError,
-            onRemovePoliciesClick,
-            toggleAddPolicyModal
           )}
+          {isFreeTier ||
+            (isPremiumTier && !selectedTeamId && selectedTeamId !== null && (
+              <p>
+                Add policies for <b>all of your hosts</b> to see which pass your
+                organization’s standards.{" "}
+              </p>
+            ))}
+        </div>
+        {updateInterval &&
+          ((selectedTeamId && !isTeamPoliciesError && !!teamPolicies?.length) ||
+            (!selectedTeamId &&
+              selectedTeamId !== null &&
+              !isGlobalPoliciesError &&
+              !!globalPolicies?.length)) && (
```

Comment:
Prime example of the comment above. I'm not sure what's happening here at first glance, so a nice, meaningful boolean variable name would be great here.